File: python/test_bubble_sort.py
# -*- coding: utf-8 -*-
''' module bubble_sort '''

import unittest
import bubble_sort


class TestBubbleSort(unittest.TestCase):
    '''bubble sort test '''

    def test_bubble_sort(self):
        '''test bubble sort'''

        before = [4, 3, 2, 1]
        after = bubble_sort.bubble_sort(before)
        self.assertEqual([1, 2, 3, 4], after, "must sort")

File: ruby/test_unit/test_bubble_sort.rb
require 'test/unit'
require_relative '../bubble_sort'

class BubbleSortTest < Test::Unit::TestCase

  def test_bubble_sort
	list = [4,3,2,1]
	BubbleSort.sort(list)
	assert_equal([1,2,3,4], list)
  end

end
